kubernetes_deploy/development/collect-static-job.yml

```diff
@@ -8,6 +8,11 @@ spec:
   template:
     metadata:
       name: laa-legal-adviser-api-collect-static
+      labels:
+        app: laa-legal-adviser-api
+        tier: collect-static
+        type: job
```

Comment:
_not a blocker, for use case/context_
For the use case of querying pods related to this application (which I think I will be doing for alerts/status), `app` would be sufficient for me.

Reply:
(_maybe_ the service area/team too)

Reply:
I think being able to select job pods could be helpful (particularly without automated culling of them), and I figured I might as well include `tier` rather than omit it, might be helpful if we ever want to look at their run times or anything 🤷‍♀

Reply:
Sorry, didn't see the 2nd message until just now.
Is that stuff valuable even with separation by namespace? Happy to add it in a future PR if it's useful

Reply:
No, not right now, but thank you for the follow-up offer.

These service area/service team labels will be useful when we start building dashboards.

The service team should be aware if anything is in an "error" state. My current interpretation is that labelling makes querying "all pods with an error state belonging to this team" possible without having to maintain a list of the namespace or application names.

Similarly, for me, as an architect, a service-area-wide "prod errors" view would be great and would also be possible without hardcoding the service team names.

Both of these points follow the "tag data" concept; we mark the interest points and parties interested in them can "subscribe" to the labels. This approach helps teams expand/change applications without coupling to the monitoring systems.

I hope this makes sense.

Reply:
💯 
I think I was forgetting querying could be done across namespaces